src/multiselect_controls.js

```diff
@@ -0,0 +1,418 @@
+/**
+ * @license
+ * Copyright 2022 MIT
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @fileoverview Multiple selection svg switch.
+ */
+
+import * as Blockly from 'blockly/core';
+
+import DragSelect from '../lib/ds.min';
+import {blockSelection, inMultipleSelectionMode,
+  setSelectionMode} from './global';
+
+/**
+ * Icon path of the multi select controls when enabled.
+ * @type {string}
+ */
+let ENABLED_IMG = 'https://github.com/mit-cml/workspace-multiselect/raw/main/test/media/select.svg';
+
+/**
+ * Icon path of the multi select controls when disabled.
+ * @type {string}
+ */
+let DISABLED_IMG = 'https://github.com/mit-cml/workspace-multiselect/raw/main/test/media/unselect.svg';
```

Comment:
I would rather these be constants and the actual values used for the plugin be stored on the plugin itself. Otherwise, this prevents multiple workspaces on the same page from having different configurations.